Index: b/_imagingcms.c
===================================================================
--- a/_imagingcms.c
+++ b/_imagingcms.c
@@ -24,7 +24,7 @@
 "
 
 #include "Python.h"
-#include "lcms.h"
+#include "lcms2.h"
 #include "Imaging.h"
 #include "py3.h"
 
Index: b/setup.py
===================================================================
--- a/setup.py
+++ b/setup.py
@@ -238,8 +238,8 @@
                 if dir:
                     _add_directory(self.compiler.include_dirs, dir, 0)
 
-        if _find_include_file(self, "lcms.h"):
-            if _find_library_file(self, "lcms"):
+        if _find_include_file(self, "lcms2.h"):
+            if _find_library_file(self, "lcms2"):
                 feature.lcms = "lcms"
 
         if _tkinter and _find_include_file(self, "tk.h"):
@@ -299,7 +299,7 @@
             if sys.platform == "win32":
                 extra.extend(["user32", "gdi32"])
             exts.append(Extension(
-                "_imagingcms", ["_imagingcms.c"], libraries=["lcms"] + extra))
+                "_imagingcms", ["_imagingcms.c"], libraries=["lcms2"] + extra))
 
         if sys.platform == "darwin":
             # locate Tcl/Tk frameworks
